src/main/java/org/broadinstitute/hellbender/tools/walkers/mutect/Mutect2FilteringEngine.java

```diff
@@ -207,10 +208,27 @@ private void applyEventDistanceFilters(final VariantContext vc, final Collection
         }
     }
 
+    // This filter checks for the case in which PCR-duplicates with unique UMIs (which we assume is caused by barcode swap)
+    // amplify the erroneous signal for an alternate allele
+    private void applyDuplicatedAltReadFilter(final M2FiltersArgumentCollection MTFAC, final VariantContext vc, final Collection<String> filters) {
+        Genotype tumorGenotype = vc.getGenotype(tumorSample);
+
+        if (!tumorGenotype.hasExtendedAttribute(UniqueAltReadCount.UNIQUE_ALT_READ_SET_COUNT_KEY)) {
+            return;
+        }
+
+        final int uniqueReadSetCount = GATKProtectedVariantContextUtils.getAttributeAsInt(tumorGenotype, UniqueAltReadCount.UNIQUE_ALT_READ_SET_COUNT_KEY, -1);
+
+        if (uniqueReadSetCount <= MTFAC.uniqueAltReadCount) {
```

Comment:
I have reservations about this. Remember that nice separation of FPs and TPs was done with a count, not fraction. A fraction may very well separate them nicely but I'd have to go back to the data and that's extra work (maybe 1-2 hours). Plus I tried to come up with a reason why using a fraction was better but I couldn't - to me thinking of the filter in terms of counts is easier to conceptualize